specs.props

```diff
@@ -14,6 +14,7 @@
     <ItemGroup>
         <!-- <PackageReference Include="Dolittle.Common.Specs" Version="$(DolittleCommonSpecsVersion)"/> -->
 
+        <PackageReference Include="coverlet.collector" Version="$(CoverletVersion)" />
```

Comment:
```suggestion
        <PackageReference Include="Coverlet.Collector" Version="$(CoverletVersion)" />
```

Reply:
It is actually all lowercased on NuGet :)